cli/docs/lacework_api.md

```diff
@@ -22,7 +22,7 @@ To list all available Lacework schema types:
 
     lacework api get /v2/schemas
 
-For a complete list of available API v1 endpoints visit:
+For a complete list of available API v2 endpoints visit:
```

Comment:
These are automatically generated docs, it will get updated with our pipeline:
```suggestion
For a complete list of available API v1 endpoints visit:
```

Reply:
Ah ok, that is good to know.
No worries on the contribution - I will endeavour to do more!